__tests__/hooks/useFeed.test.js

```diff
@@ -182,7 +182,7 @@ describe('Logic feed should run correctly', () => {
   it('initialSetup should run correctly', async () => {
     const {result} = renderHook(useFeed);
     await result.current.setTotalVote(1);
-    expect(result.current.handleTextCountStyle()).toEqual('#107793');
+    expect(result.current.handleTextCountStyle()).toEqual('#00ADB5');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color code has been changed from `#107793` to `#00ADB5`. Ensure that this change is intentional and aligns with the overall color scheme of your application. If it's a mistake, revert it back.
```diff
-    expect(result.current.handleTextCountStyle()).toEqual('#00ADB5');
+    expect(result.current.handleTextCountStyle()).toEqual('#107793');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->